scripts/qa_crowbarsetup.sh

```diff
@@ -4734,6 +4735,11 @@ function onadmin_crowbar_nodeupgrade
             # Do not use it until https://bugzilla.novell.com/show_bug.cgi?id=997293 is fixed
             # safely crowbarctl upgrade services
             curl -s -X POST -H "$crowbar_api_v2_header" $crowbar_api/api/upgrade/services
+
+            if grep -q "failed" /var/lib/crowbar/upgrade/6-to-7-progress.yml ; then
+                complain 12 "Services step has failed. Check the upgrade status."
```

Comment:
'Check the upgrade status' could give a hint to how to do that - or maybe do it here?